bookwyrm/views/shelf/shelf.py

```diff
@@ -103,6 +112,7 @@ def get(self, request, username, shelf_identifier=None):
             "page_range": paginated.get_elided_page_range(
                 page.number, on_each_side=2, on_ends=1
             ),
+            "shelves_filter_query": shelves_filter_query,
```

Comment:
Sorry this is really pedantic but the "Filters" label looks kind of goofy to me because it's a level 5 `title` in the filters_panel template. Can we add `"size": "small",` here to make it a bit smaller?

Reply:
Cool with me. Although that is a very ambiguously named value to pass in from a view model 😅 . Took me a minute to figure out how "small" was used.

Reply:
yeah i think it's generally passed within the templates but I couldn't work out how you would do it when `include`ing - you effectively want to pass it up to the parent.